src/test/components/MenuButtons.test.js

```diff
@@ -144,7 +143,7 @@ describe('app/MenuButtons', function() {
       if ((window: any).TextEncoder) {
         throw new Error('A TextEncoder was already on the window object.');
       }
-      (window: any).TextEncoder = TextEncoder;
+      (window: any).TextEncoder = new TextEncoder();
```

Comment:
I think you can just remove this line, with TextEncoder being global?
Not sure :-)

If that doesn't work, this is maybe because we use a jsdom environment. In that case you can keep this line as it was and just remove the import at the top of the file.
Lastly if this still doesn't work, just forget about this part.

Reply:
The problem with TextDecoder/TextEncoder is because jsdom doesn't support them, see: https://github.com/jsdom/jsdom/issues/2524

Because we use the jsdom environment for our tests, we don't have them as globals :( So I'd suggest removing this change at all, sadly.

Reply:
Hi @julienw ,

I came across that issue with JSDOM and TextEncoder/TextDecoder also and was just about to mention it. Looks like it will have to be left as it is for now! It did throw me for a while when nothing I tried was working!